src/inspect_ai/_view/www/src/state/hooks.ts

```diff
@@ -167,7 +173,7 @@ export const useFilteredSamples = () => {
 
     // Sort samples
     const sorted = samplesDescriptor
-      ? sortSamples(sort, filtered, samplesDescriptor, score)
+      ? sortSamples(sort, filtered, samplesDescriptor, selectedScores?.[0])
```

Comment:
Should sorting accept the list and subsort by scores in order when showing multiple?